sig-scalability/README.md

```diff
@@ -49,6 +49,10 @@ subprojects, and resolve cross-subproject technical issues and decisions.
 ## Subprojects
 
 The following [subprojects][subproject-definition] are owned by sig-scalability:
+### inference-perf
```

Comment:
nit: let's move it to the end of subprojects list, as the newest one

Reply:
Should this be alphabetically?

Reply:
@wojtek-t the generator will place it at the top because of alphabetic order